src/main/java/server/HTTProtocol.java

```diff
@@ -0,0 +1,30 @@
+package server;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+import static server.StatusCodes.*;
```

Comment:
I'd recommend avoiding using wildcard imports. It can lead to ambiguous imports, but more importantly, it's less clear where your classes are coming from.

https://google.github.io/styleguide/javaguide.html#s3.3-import-statements